test/org/loklak/api/search/QuoraProfileScraperTest.java

```diff
@@ -0,0 +1,130 @@
+            profileListTimeLine = (Timeline2)executePrivateMethod(QuoraProfileScraper.class, quoraScraper, "scrapeProfile", new Class[]{BufferedReader.class, String.class}, br, url);
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+        JSONArray profileList = profileListTimeLine.toArray();
+        JSONObject quoraProfile = (JSONObject)profileList.get(0);
+
+        assertEquals(quoraProfile.getString("search_url"), url);
+        assertEquals(quoraProfile.getString("user"), userName);
+        assertEquals(quoraProfile.getString("rss_feed_link"), rssFeedLink);
+        assertEquals(quoraProfile.getString("profileImage"), profileImage);
+        assertEquals(quoraProfile.getJSONObject("feeds").getString("topics_url"), topicsUrl);
+        assertEquals(quoraProfile.getJSONObject("feeds").getString("following_url"), followingUrl);
+        assertEquals(quoraProfile.getJSONObject("feeds").getString("blogs_url"), blogsUrl);
+        assertEquals(quoraProfile.getJSONObject("feeds").getString("edits_url"), editsUrl);
+        assertEquals(quoraProfile.getJSONObject("feeds").getString("posts_url"), postsUrl);
+        assertEquals(quoraProfile.getJSONObject("feeds").getString("questions_url"), questionsUrl);
+    }
+
+    @Test
+    public void quoraQuestionScraperUserTest() {
+
+        QuoraProfileScraper quoraScraper = new QuoraProfileScraper();
+        Timeline2 questionListTimeLine = null;
+        String url = "https://www.quora.com/search/?q=fossasia&type=question";
+        String postType = "question";
+        BufferedReader br = null;
+                
+        try {
+            ClientConnection connection = new ClientConnection(url);
+            //Check Network issue
+            assertThat(connection.getStatusCode(), is(200));
+
+            br = quoraScraper.getHtml(connection);
+        } catch (IOException e) {
+            DAO.log("GithubProfileScraperTest.githubProfileScraperUserTest() failed to connect to network. url:" + url);
+        }
+
+        try {
+            questionListTimeLine = (Timeline2)executePrivateMethod(QuoraProfileScraper.class, quoraScraper, "scrapeQues", new Class[]{BufferedReader.class, String.class}, br, url);
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+        JSONArray qList = questionListTimeLine.toArray();
+        
+        assertFalse(qList.length() == 0);
+
+        for (int i = 0; i < qList.length(); i++) {
+            JSONObject question = (JSONObject)qList.get(i);
+            assertEquals(question.getString("post_type"), postType);
+            assertEquals(question.getString("search_url"), url);
+        } 
+    }
+
+    public Object executePrivateMethod(
+            Class<?> clazz,
+            Object instanceObj,
+            String methodName,
+            Class<?>[] parameterTypes,
+            Object ... args
+    ) throws Exception {
+
+        // Get declared Method for execution
```

Comment:
@vibhcool could you please explain a bit. Are you suggesting to move methods using reflection api in a separate class or interface and then use them in Twitter and Quora tests?